custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -385,7 +387,7 @@ def pre_aggregation_queries(self):
 
     def create_temporary_table(self):
         return """
-        CREATE UNLOGGED TABLE \"{table}\" (LIKE child_health_monthly) PARTITION BY LIST (state_id);
+        CREATE TABLE \"{table}\" (LIKE child_health_monthly) PARTITION BY LIST (state_id);
```

Comment:
What are the motivations and implications of this change?

Reply:
This was a temporary table before (agg was insert into temp table -> copy from temp to single child health monthly table). The temp table was unlogged to increase that insert performance.

The new agg will just be insert into this table and then attach it as a partition. Since it will be used as a real table (and replicated to the standbys) it needs to be a logged table. This change will increase the duration of the insertion stage, but since we remove the copy at the end it should still decrease the overall time.